modal/cli/volume.py

```diff
@@ -143,30 +142,30 @@ async def ls(
             raise UsageError(exc.message)
         raise
 
-    if sys.stdout.isatty():
-        console = Console()
-        console.print(f"Directory listing of '{path}' in '{volume_name}'")
-        table = Table()
-        for name in ["filename", "type", "created/modified", "size"]:
-            table.add_column(name)
-
+    if not json and not sys.stdout.isatty():
+        # Legacy behavior -- I am not sure why exactly we did this originally but I don't want to break it
```

Comment:
Also have no context on this. From a scripting perspective it's nice that we omit the table formatting, but this seems oddly terse!